assets/tests-helper/functions.js

```diff
@@ -0,0 +1,8 @@
+export const mockSearch = ( search ) => {
```

Comment:
I created this folder to add tests helper. As this grows, we can rethink this structure of helpers.
Or if someone has a strong opinion about this, we could do something right now.